includes/class-frontend.php

```diff
@@ -53,7 +53,7 @@ public function show_warranty() {
                     $duration   = wcrw_get_duration_value( $warranty['length_duration'], $value );
                     echo '<p class="wcrw_warranty_info"><strong>'. $warranty_label .':</strong> '. $value .' '. $duration .'</p>';
                 } else {
-                    echo '<p class="wcrw_warranty_info"><strong>'. $warranty_label .':</strong> '. __( 'Lifetime', 'wc-return-warranty' ) .'</p>';
+                    echo '<p class="wcrw_warranty_info"><strong>'. "hello" .$warranty_label .':</strong> '. __( 'Lifetime', 'wc-return-warranty' ) .'</p>';
```

Comment:
What is `hello` here ?

Reply:
I forgot  to remove this . I used this for testing output